libc-top-half/musl/include/sched.h

```diff
@@ -38,6 +38,10 @@ int    sched_getscheduler(pid_t);
 int    sched_rr_get_interval(pid_t, struct timespec *);
 int    sched_setparam(pid_t, const struct sched_param *);
 int    sched_setscheduler(pid_t, int, const struct sched_param *);
+#else /* ... but this struct isn't optional. */
+struct sched_param {
+	int sched_priority;
+};
```

Comment:
Should we just use the struct above (i.e. shrink the `ifdef` region above)?

Reply:
Done, except I decided to make two separate ifdef regions instead of putting a single close bracket inside a `#else` region (which looked ugly)

Reply:
On second thought, decided to simply keep all of the reserved fields in the struct, `#ifdef`-ing out only the functions. This seems to be what you originally meant?

Reply:
Yup!